string collision_1
string collision_2
float32[3] normal
float32[3] position
float32[3] forces
float32 depth
